test/CMakeLists.txt

```diff
@@ -48,5 +38,33 @@ add_custom_target(installcheck
   TEST_INPUT_DIR=${TEST_INPUT_DIR}
   TEST_OUTPUT_DIR=${TEST_OUTPUT_DIR}
   PG_REGRESS=${PG_REGRESS}
-  ${CMAKE_CURRENT_SOURCE_DIR}/pg_regress.sh ${PG_REGRESS_OPTS}
+  )
+
+set(PG_REGRESS_OPTS_TEMP_INSTANCE
+  --user=${TEST_PGUSER}
+  --port=${TEST_PGPORT_TEMP_INSTANCE}
+  --temp-instance=${TEST_CLUSTER}
+  --temp-config=${TEST_OUTPUT_DIR}/postgresql.conf
+  ${PG_REGRESS_OPTS_COMMON}
+)
+
+set(PG_REGRESS_OPTS_LOCAL
+  --user=postgres
+  --port=${TEST_PGPORT_LOCAL}
+  ${PG_REGRESS_OPTS_COMMON}
+)
+
+# Configuration setting needed to LOAD module for tests instead of
```

Comment:
Actually, noted that this comments is outdated (not your fault). But since you are messing with this we should probably also update/remove the comment.

Reply:
fixed